.meta.toml

```
@@ -0,0 +1,15 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/buildout-recipe
+[meta]
+template = "buildout-recipe"
+commit-id = "519ffffe1b360ede58b32a375b5f8b26059d8016"
+
+[python]
+with-appveyor = false
+with-pypy = true
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[coverage]
+fail-under = 19
```

Coveralls shows 20% maybe we can up it a bit here?
```suggestion
fail-under = 20
```

---

I do not know where `coveralls` gets this number from. The `coverage` tox environment yields `19.54 %` coverage. So it will fail if we require 20.